dlrover/python/elastic_agent/config/paral_config_tuner.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 The DLRover Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import threading
+import time
+
+from dlrover.python.common.constants import ConfigPath
+from dlrover.python.common.grpc import ParallelConfig
+from dlrover.python.common.singleton import singleton
+from dlrover.python.elastic_agent.master_client import GlobalMasterClient
+
+
+@singleton
+class ParalConfigTuner(object):
+    def __init__(self):
+        """
+        Parallelism config tuner for updating parallelism config file.
+        """
+        self._master_client = GlobalMasterClient.MASTER_CLIENT
+        self.config_dir = os.path.dirname(
+            os.environ[ConfigPath.ENV_PARAL_CONFIG]
+        )
+        self.config_path = os.environ[ConfigPath.ENV_PARAL_CONFIG]
```

Comment:
```
self.config_path = os.getenv(ConfigPath.ENV_PARAL_CONFIG, "")
```
We can avoid the error if the ConfigPath.ENV_PARAL_CONFIG is not in `os.environ`.